test/spec/pathToRegExpWrap.spec.js

```diff
@@ -7,22 +7,89 @@
 
 "use strict";
 
-const path = require('../../')();    // options object could be passed as well 
+const path = require('../../')();    // options object could be passed as well
 
 describe('Testing path-to-regexp-wrap', () => {
-	let route, match;
-	
-	beforeEach(() => {
-		route = '/get/:id';
-  	match = path(route);
-	});
-
-	afterEach(() => {
-		route = match = null;
-	})
-
-	it('Should match the path and return parameters object', () => {
-		assert.deepEqual(match('/get/1'), {id: '1'}, 'Should return matched parameters object');
-	});
-
-});
\ No newline at end of file
+  let route, match;
+
+  beforeEach(() => {
+    route = '/get/:id';
+    match = path(route);
+  });
+
+  afterEach(() => {
+    route = match = null;
+  });
+
+  it('correct footprint', () => {
+    expect(path('/')).to.be.an.instanceof(Function);
+    expect(path('/')('/')).to.be.an.instanceof(Object);
+    expect(path('/')('/bla')).to.equal(false);
+  });
+
+  it('Should match the path and return parameters object', () => {
+    assert.deepEqual(match('/get/1'), {id: '1'}, 'Should return matched parameters object');
```

Comment:
I think we should probably replace `assert` with BDD's` expect` here. Just for consistency.